codegen_glibmm/dbustypes.py

```diff
@@ -34,6 +34,18 @@ def __init__(self, lines):
         self.lines = lines
 
 
+class Error:
+    def __init__(self, value):
+        (prefix, name) = value.rsplit('.', 1)
```

Comment:
What if the interface write choses a non-dotted error name? like (in the XML) `value="MyError"` ?